frontend/src/views/FormContainer.tsx

```diff
@@ -10,28 +10,44 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-import { Banner } from 'core/banner/Banner';
-import { Text } from 'core/text/Text';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
 import PropTypes from 'prop-types';
 import React from 'react';
 import styles from './FormContainer.module.css';
 
 const propTypes = {
   title: PropTypes.string.isRequired,
   subtitle: PropTypes.string.isRequired,
-  banner: PropTypes.node,
   children: PropTypes.node.isRequired,
 };
 
-export const FormContainer = ({ title, subtitle, banner, children }) => {
-  let bannerContent = banner ? <Banner data-testid="banner" content={banner} /> : null;
+const useUploadProjectViewStyles = makeStyles((theme) => ({
+  title: {
+    fontSize: '32px',
```

Comment:
Is this normal to "hard code" the size here instead of refering to a variable as before? It could be a JS variable/constant if we do CSS-in-JS, but it seems wrong to hard code "magic values" (sizes, colors) here.

Reply:
It should be something like `theme.spacing(2)` and there shouldn't be any references to font weight or color either. None of the CSS visible here should exist here.

Reply:
done